### PR TITLE
feat(query): placeholder

### PR DESCRIPTION
**Pull request by ElisePatrikainen**

### CODE REVIEW

File: src/use-query.ts [hunk 1]
```diff
@@ -200,6 +200,15 @@ export function useQuery<TResult, TError = ErrorDefault>(
     entry,
     (entry, previousEntry) => {
       if (!isActive) return
+      if (entry.stale && entry.options?.placeholderData && previousEntry?.state.value.status === 'success') {
+        cacheEntries.setEntryState(entry, {
+          // TODO: pending?
```
Comment (posva) [inline src/use-query.ts]: the placeholder data cannot change the cache. It should only appear as a fallback while the entry is not in success state

Comment (ElisePatrikainen) [reply src/use-query.ts]: Understood, this makes sens. I changed the approach regarding it (to store the placeholder data in the entry cache but not in the entry state, as I had done first), let me know what you think about it.

The placeholder fallback should also appear in error state, right?

Comment (posva) [reply src/use-query.ts]: I think so. The point is to have fallback data while it’s undefined

Comment (ElisePatrikainen) [reply src/use-query.ts]: Perfect.

**Comment by posva** [comment]
I pushed to feat/placeholder-data. I think it will be a better starting point. Could you port the tests to that branch, push there and open a PR with it?

**Comment by ElisePatrikainen** [comment]
> I pushed to feat/placeholder-data. I think it will be a better starting point. Could you port the tests to that branch, push there and open a PR with it?

Sure, here it is: https://github.com/posva/pinia-colada/pull/80

**Pull request closed without merging**